ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5913,6 +5914,27 @@ kong_apis:
     config.required: false
     config.enabled: false
 
+- name: assessmentItemRead
+  uris: "{{ assessment_prefix }}/v1/items/read"
+  upstream_url: "{{ learning_service_url }}/assessment/v3/items/read"
+  strip_uri: true
+  plugins:
+  - name: jwt
+  - name: cors
+  - "{{ statsd_pulgin }}"
+  - name: acl
+    config.whitelist:
+    - contentAccess
```

Comment:
Can we use `assessmentAccess`?

Reply:
@keshavprasadms Same as above [comment](https://github.com/project-sunbird/sunbird-devops/pull/3339/files#r877787145).

cc: @AmiableAnil

Reply:
Now why would we want to mix content with assessment ACLs. Since these target a different category of API, we should start to use a different nomenclature.

Reply:
@keshavprasadms Updated `config.whitelist` value as suggested.